prompt_toolkit/key_binding/bindings/emacs.py

```diff
@@ -269,27 +271,6 @@ def _copy(event: E) -> None:
         data = event.current_buffer.copy_selection()
         event.app.clipboard.set_data(data)
 
-    @handle("escape", "left")
```

Comment:
could be mistaken, but these functions seem the same as `backward-word` and `forward-word`?

Reply:
It looks like you're right for ``backward-word``.

For ``forward-word`` the implementation is not the same. Are you sure they are doing the same thing? I mean, one does ``find_next_word_ending`` and the other does ``find_next_word_beginning``.

I'm not sure what is right. I'm all for less code duplication and doing the same thing GNU readline or Emacs is doing.

Reply:
great point! the [docs](https://www.gnu.org/software/emacs/manual/html_node/emacs/Words.html) were a tad hard to understand this particular part. so i downloaded emacs to see for myself. What's @register-ed in "named_commands" was right. specifically:

- `backward-word` goes to the start of the last word
- `forward-word` goes to the next non-letter-or-number (space, punctuation, etc)